python/lib/evaluations/dmod/evaluations/specification/base.py

```diff
@@ -572,36 +592,38 @@ def identities_match(self, configuration: typing.Union[dict, "Specification"]) -
             return self.name is not None and self.name == configuration.get("name")
 
     def __getitem__(self, key: str) -> typing.Any:
-        return self.__properties[key]
+        return self.properties[key]
 
     def __contains__(self, key: str) -> bool:
-        return key in self.__properties
+        return key in self.properties
 
     @abc.abstractmethod
```

Comment:
That's still the default behavior needed across all the other implementations. It makes sense to override in `ThresholdSpecification` since it is a collection of items that may be identified by name, but not so much on other types whose extra values need to be interpreted or retrieved in a dict-like fashion.